wandb/proto/wandb_internal.proto

```diff
@@ -156,6 +156,7 @@ message SettingsItem {
  */
 message HistoryRecord {
   repeated HistoryItem item = 1;
+  string step = 2;
```

Comment:
this should be an integer

Reply:
Integers have a default value of 0 passed in, so if its an integer I couldn't tell if it was actually given a step of 0, or if it was the default value

Reply:
I think HasField() is what you are looking for (pretty sure that still works with int fields in proto3)

Reply:
Thanks!

Reply:
I'm going to use a oneof here based on this conversation: https://github.com/googleapis/google-cloud-python/issues/1402

And this error I received: `ValueError: Can't test non-optional, non-submessage field "HistoryRecord.step" for presence in proto3.`

They suggest oneof as a solution further down in the conversation.